gradle/libs.versions.toml

```diff
@@ -22,32 +22,32 @@ quiltflower = "1.9.0"
-jackson-xml = { module = "com.fasterxml.jackson.dataformat:jackson-dataformat-xml", version.ref = "jackson-xml" }
-launchermeta-parser = { module = "org.quiltmc:launchermeta-parser", version.ref = "launchermeta-parser" }
+jackson_xml = { module = "com.fasterxml.jackson.dataformat:jackson-dataformat-xml", version.ref = "jackson_xml" }
+launchermeta_parser = { module = "org.quiltmc:launchermeta-parser", version.ref = "launchermeta_parser" }
 
 enigma = { module = "cuchaz:enigma", version.ref = "enigma" }
-enigma-cli = { module = "cuchaz:enigma-cli", version.ref = "enigma" }
-enigma-swing = { module = "cuchaz:enigma-swing", version.ref = "enigma" }
-enigma-plugin = { module = "org.quiltmc:quilt-enigma-plugin", version.ref = "enigma-plugin" }
-tiny-remapper = { module = "org.quiltmc:tiny-remapper", version.ref = "tiny-remapper" }
+enigma_cli = { module = "cuchaz:enigma-cli", version.ref = "enigma" }
+enigma_swing = { module = "cuchaz:enigma-swing", version.ref = "enigma" }
+enigma_plugin = { module = "org.quiltmc:quilt-enigma-plugin", version.ref = "enigma_plugin" }
+tiny-remapper = { module = "org.quiltmc:tiny-remapper", version.ref = "tiny_remapper" }
 stitch = { module = "net.fabricmc:stitch", version.ref = "stitch" }
 unpick = { module = "org.quiltmc.unpick:unpick", version.ref = "unpick" }
-unpick-cli = { module = "org.quiltmc.unpick:unpick-cli", version.ref = "unpick" }
-unpick-format-utils = { module = "org.quiltmc.unpick:unpick-format-utils", version.ref = "unpick" }
-mapping-io = { module = "net.fabricmc:mapping-io", version.ref = "mapping-io" }
-javadoc-draftsman = { module = "org.quiltmc:javadoc-draftsman", version.ref = "javadoc-draftsman" }
+unpick_cli = { module = "org.quiltmc.unpick:unpick-cli", version.ref = "unpick" }
+unpick_format_utils = { module = "org.quiltmc.unpick:unpick-format-utils", version.ref = "unpick" }
+mapping_io = { module = "net.fabricmc:mapping-io", version.ref = "mapping_io" }
+javadoc_draftsman = { module = "org.quiltmc:javadoc-draftsman", version.ref = "javadoc_draftsman" }
 
-fabric-loader = { module = "net.fabricmc:fabric-loader", version.ref = "fabric-loader" }
-jetbrains-annotations = { module = "org.jetbrains:annotations", version.ref = "jetbrains-annotations" }
+fabric_loader = { module = "net.fabricmc:fabric-loader", version.ref = "fabric_loader" }
+jetbrains_annotations = { module = "org.jetbrains:annotations", version.ref = "jetbrains_annotations" }
 jsr305 = { module = "com.google.code.findbugs:jsr305", version.ref = "jsr305" }
 
 # Decompilers
 cfr = { module = "net.fabricmc:cfr", version.ref = "cfr" }
 quiltflower = { module = "org.quiltmc:quiltflower", version.ref = "quiltflower" }
 
 [bundles]
-enigma-full = ["enigma", "enigma-cli", "enigma-swing", "enigma-plugin"]
-unpick-full = ["unpick", "unpick-cli", "unpick-format-utils"]
\ No newline at end of file
+enigma_full = ["enigma", "enigma_cli", "enigma_swing", "enigma_plugin"]
+unpick_full = ["unpick", "unpick_cli", "unpick_format_utils"]
```

Comment:
Could you add a new line at the end of the file?

Reply:
done!